upup/pkg/fi/cloudup/alitasks/loadbalancer.go

```diff
@@ -73,11 +74,15 @@ func (l *LoadBalancer) Find(c *fi.Context) (*LoadBalancer, error) {
 
 	klog.V(2).Infof("found matching LoadBalancer: %q", *l.Name)
 
-	actual := &LoadBalancer{}
-	actual.Name = fi.String(responseLoadBalancers[0].LoadBalancerName)
-	actual.AddressType = fi.String(string(responseLoadBalancers[0].AddressType))
-	actual.LoadbalancerId = fi.String(responseLoadBalancers[0].LoadBalancerId)
-	actual.LoadBalancerAddress = fi.String(responseLoadBalancers[0].Address)
+	lb := responseLoadBalancers[0]
```

Comment:
Same as @zetaab's comment above, we should check the length of the list here and error if it isnt what we expect.

Reply:
Hi Peter, thanks for reviewing this. Actually the length of this list is already checked in the existing codes, as below:

https://github.com/kubernetes/kops/blob/701375cd897ed891e5e76523d0fd928c5c8f529d/upup/pkg/fi/cloudup/alitasks/loadbalancer.go#L68-L77

But this reminds me that I can add some logging when there is no LoadBalancer were found. 😄